### DISCUSSION TITLE
노션 글 작성 시 자동 반영에 대한 질문이 있습니다.

### DISCUSSION THREAD

**Discussion by jongeuni** [morethanmin/morethan-log#187] [Q&A]
안녕하세요 :) 만들어주신 오픈 소스 흥미롭게 보고, 현재 적용 단계입니다

<img width="877" alt="image" src="https://github.com/morethanmin/morethan-log/assets/66874658/3265d550-9105-4eee-8deb-ad59c3aa7b86">
작성해주신 리드미 파일에서, 
노션에서 작성된 게시물은 사이트에서 자동으로 업데이트 된다고 설명이 써져 있는데요.

현재 vercel에서 재배포를 눌러주거나, github에서 변경을 해주는 게 아니면 업데이트가 안 되는 것 같아서 질문 드립니다.

노션 게시물 자동 업데이트를 위해
혹시 따로 설정해주어야 하는 게 있을까요?

NOTION_PAGE_ID는 추가하였습니다!

**Comment by morethanmin** [comment]
따로 설정이 필요한건 없습니다! 다만 get static props 특성상 어느정도의 딜레이는 발생합니다. (대략 10초 정도..? 였던 것 같습니다)
